Sensor options (#229)
levoai/docs: docs/install-traffic-capture-sensors/ebpf-sensor/sensor-kubernetes.md

```bash
# Replace 'hostname|IP' & 'port' with the values you noted down from the Satellite install
# If Sensor is installed on same cluster as Satellite, use 'levoai-haproxy'

> # If they are installed on different clusters, the haproxy should be exposed so that it is
> # reachable to the sensor, mention the exposed ip-address as satellite-url

Reviewer: ```suggestion
# If they are installed on different clusters, the haproxy service should be exposed so that it is
# reachable by the sensor. Use the exposed address as the value for satellite-url.
```

  --create-namespace \
  --set sensor.config.default-service-name=<'Application Name' chosen earlier> \
  --set sensor.config.satellite-url=<hostname|IP:port> \
<<<<<<< HEAD
  --set sensor.levoaiEnvironment=<'Application environment'>
=======
  --set sensor.config.organization-id=<your-org-id> \
  --set sensor.config.env=<'Application environment'>

> >>>>>>> origin

Reviewer 2: fix conflicts

  --create-namespace \
  --set sensor.config.default-service-name=<'Application Name' chosen earlier> \
  --set sensor.config.satellite-url=<hostname|IP:port> \

>   --set sensor.config.organization-id=<your-org-id> \

Reviewer: Org ID should still be set.

  --create-namespace \
  --set sensor.config.default-service-name=<'Application Name' chosen earlier> \
  --set sensor.config.satellite-url=<hostname|IP:port> \

>   --set sensor.config.organization-id=<your-org-id> \ \

Reviewer: Remove the extra slash
```suggestion
  --set sensor.config.organization-id=<your-org-id> \
```

#### NOTE:
The default address for the satellite url in helm installations is `levoai-haproxy`.
This address assumes that the Satellite is installed in the same cluster (and namespace) as the Sensor.

> If they are installed on different clusters, the haproxy should be exposed so that it is
> reachable to the sensor, mention the exposed ip-address as satellite-url

Reviewer: ```suggestion
If they are installed on different clusters, the haproxy service should be exposed so that it is
reachable by the sensor. Use the exposed address as the value for satellite-url.
```
